### PR TITLE
Added usage guide to run CrateDB Jepsen tests

### PR DESCRIPTION
**Pull request by matriv**
Please also check the layout: [here](https://github.com/crate/jepsen/tree/mt/add-readme-usage/crate)

### CODE REVIEW

File: crate/README.md [hunk 1]
```diff
@@ -1,10 +1,76 @@
...
+example: ``20171018T150316.000Z`` and also a symlink ``latest`` that points to
+the latest run of the test.
+
+Inside each individual run of a test you can find the log files and results of
+the test run. More specifically you can see:
+
+ - ``jepsen.log`` containing logging information of the test run
+ - ``timeline.txt`` graphical timeline representation of the events that took 
+   place. 
+ - ``latency-quantiles.png``
```
Comment (matriv) [inline crate/README.md]: @andrei @seut Would need some help to explain those 3 images.

Comment (matriv) [inline crate/README.md]: @andreidan @seut Would need some help to explain those 3 images.

**Review by andreidan — APPROVED** [review]
lgtm

File: crate/README.md [hunk 2]
```diff
@@ -62,12 +62,11 @@ the latest run of the test.
 Inside each individual run of a test you can find the log files and results of
 the test run. More specifically you can see:
 
- - ``jepsen.log`` containing logging information of the test run
+ - ``jepsen.log`` containing logging information of the test run.
```
Comment (andreidan) [inline crate/README.md]: "contains"

File: crate/README.md [hunk 3]
```diff
@@ -62,12 +62,11 @@ the latest run of the test.
...
  - ``timeline.txt`` graphical timeline representation of the events that took 
    place. 
- - ``latency-quantiles.png``
- - ``latency-raw.png``
- - ``rate.png``
+ - ``latency-quantiles.png``, ``latency-raw.png``, ``rate.png`` various graphs
+   related to the events that took place during the test run.
  - ``n[1-5]`` directories each one containing the ``crate.log`` file of each of
```
Comment (andreidan) [inline crate/README.md]: maybe this is a bit more clear "a directory for every crate node configured for the test, each containing the crate log"

File: crate/README.md [hunk 4]
```diff
@@ -1,10 +1,80 @@
 # jepsen.crate
 
-A Clojure library designed to ... well, that part is up to you.
+A Clojure library designed to test CrateDB for the following issues that can 
+occur because of network paritioning:
```
Comment (seut) [inline crate/README.md]: typo: paritioning

**Pull request merged**